jbmc/src/java_bytecode/java_class_loader.cpp

```diff
@@ -116,6 +116,16 @@ java_class_loadert::get_parse_tree(
   parse_tree_with_overlayst &parse_trees = class_map[class_name];
   PRECONDITION(parse_trees.empty());
 
+  // do we refuse to load?
+  if(!class_loader_limit.load_class_file(class_name_to_file(class_name)))
+  {
+    debug() << "not loading " << class_name << " because of limit" << eom;
+    java_bytecode_parse_treet parse_tree;
+    parse_tree.parsed_class.name=class_name;
+    parse_trees.push_back(std::move(parse_tree));
+    return parse_trees;
+  }
+
```

Comment:
This duplicates the code at the end of the function, but ok with me.
nitpick ` = ` in line 124

Reply:
Yes, understood, did bug me as well.